### PR TITLE
Clarify the versions

### PR DESCRIPTION
**Pull request by bricelam**
Part of #3

### CODE REVIEW

**Comment by utelle** [comment]
Looks good. Thanks.

Additionally we could use annotated tags when tagging the repo for a release. I use that approach for _SQLite3 Multiple Ciphers_ itself, where I add the version information of SQLite in the annotation.

**Comment by utelle** [comment]
Please note that [SQLite3 Multiple Ciphers](https://github.com/utelle/SQLite3MultipleCiphers) was updated today based on _SQLite version 3.44.0_ which was released a few days ago. 

The library binaries for use with the NuGet packages were rebuilt, too. See [SQLite3MultipleCiphers-cb](https://github.com/utelle/SQLite3MultipleCiphers-cb/tree/sqlite3mc).

That is, the current version numbers are now:

- SQLite3 Multiple Ciphers **1.7.3**
- SQLite **3.44.0**

**Comment by bricelam** [comment]
Updated. Also addressed the things mentioned in #4

File: cb
Comment (bricelam) [inline cb:1]: Deleting this because I think I added it by accident. If you added it intentionally, I'll revert.

Comment (utelle) [reply cb:1]: > Deleting this "submodule" because I think I added it by accident. If you added it intentionally, I'll revert.

No, I didn't add this.

**Review by utelle — APPROVED** [review]
This looks really good. A single place to update the version numbers. Thanks.

**Pull request merged**